cluster/manifests/admission-control/teapot.yaml

```diff
@@ -14,7 +14,7 @@ webhooks:
       - operations: [ "CREATE" ]
         apiGroups: [""]
         apiVersions: ["v1"]
-        resources: ["pods"]
+        resources: ["pods", "pods/binding"]
```

Comment:
Please split it into a separate rule, with `failurePolicy: Ignore`.

Reply:
good idea - does this mean we could propagate the errors from binding (again)?
Might make troubleshooting easier.

Reply:
anyways, I've changed as advised.